### PR TITLE
linters: enable errcheck

### PR DESCRIPTION
**Pull request by kyessenov**
/assign @geeknoid 
Signed-off-by: Kuat Yessenov <kuat@google.com>

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -153,6 +153,12 @@ issues:
   # excluded by default patterns execute `golangci-lint run --help`
   exclude: []
 
+  exclude-rules:
+    # Exclude some linters from running on tests files.
+    - path: _test\.go$|^tests/|^samples/
+      linters:
+        - errcheck
```
Comment (geeknoid) [inline .golangci.yml]: Oh man, I didn't know this option was available. Could you disable maligned on test as well please?

File: mixer/adapter/stackdriver/contextgraph/send.go [hunk 2]
```diff
@@ -182,7 +182,7 @@ func (h *handler) call(ctx context.Context, req *contextgraphpb.AssertBatchReque
 		s, _ := status.FromError(err)
 		if d := s.Proto().Details; len(d) > 0 {
 			// Log the debug message, if present.
-			h.env.Logger().Errorf("STATUS: %s\n", d[0].Value)
+			_ = h.env.Logger().Errorf("STATUS: %s\n", d[0].Value)
```
Comment (geeknoid) [inline mixer/adapter/stackdriver/contextgraph/send.go]: Seems like should be getting logged with Debugf perhaps?

File: .golangci.yml (hunk 1 above)
Comment (kyessenov) [reply .golangci.yml]: Done

File: mixer/adapter/stackdriver/contextgraph/send.go (hunk 2 above)
Comment (kyessenov) [reply mixer/adapter/stackdriver/contextgraph/send.go]: I don't know... I left it as it is since not sure what the purpose of this log line is.

**Comment by geeknoid** [comment]
Try try again...

/retest

**Pull request merged**